server/src/routers/MenusRouter/getMenus.js

```diff
@@ -1,12 +1,37 @@
-import getOneMenu from "./getOneMenu";
-import getAllMenus from "./getAllMenus";
+import firestore from "../../config/firebase/firebaseConfig";
+
+import diningLocations from "../../config/locations";
+import * as constants from "../../config/constants";
 
 export default async function getMenus(query) {
-    if (!("location" in query) || query.location == "all") {
-        const menus = await getAllMenus(query);
-        return menus;
-    } else {
-        const menu = await getOneMenu(query);
-        return menu;
+	const locations = (!query.location || query.location == "all") 
+		? Object.values(diningLocations) 
+		: [diningLocations[query.location]];
+	console.log(`Locations ${locations}`)
+
+	let todayDoc = undefined;
+    let tomorrowDoc = undefined;
+    try {
+        todayDoc = await firestore.doc("menus/today").get();
+        tomorrowDoc = await firestore.doc("menus/tomorrow").get();
+    } catch (e) {
+        throw new Error(constants.E_DB_READ + e);
```

Comment:
I think we should `console.error` and fall back to the old route in this case, rather than throwing.